src/services/datahub-client.ts

```diff
@@ -199,23 +205,25 @@ export class HttpDatahubClient implements DatahubClient {
     }
   }
 
-  public async getDailyVisitsByPeriod({
+  public async getDailyAndHoursVisitsByPeriod({
```

Comment:
What do you think about a name more similar to DataHub API, for example: `getVisitsQuantitySummarizedByPeriod`?

Reply:
oh y didnt see that. Yes i can change. its more clear